README.md

```diff
@@ -48,13 +48,8 @@ I'm not far from completing [milestone 0.1.0](https://github.com/buzz-language/b
 cd mir
 make
 ```
-4. Add to your shell rc:
-```bash
-export BUZZ_PATH="/path/to/buzz/dist"
-export PATH="$BUZZ_PATH/bin:$PATH"
```

Comment:
I think we should still advice to put buzz binary in the `PATH`

Reply:
yes